# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/80_mm-2.7-strip.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/Makefile mm2.7/src/Makefile
--- mm2.7.orig/src/Makefile	2003-05-15 00:11:18.000000000 +0200
+++ mm2.7/src/Makefile	2003-05-15 00:19:21.000000000 +0200
@@ -72,7 +72,7 @@
 #	install -s -c $(LOCALBINDIR) $$file
 #
 #INSTALL = cp
-INSTALL = install -c -s
+INSTALL = install -c
 
 # Root of installation tree
 INSTROOT = /usr
